.github/workflows/frontend_publish_release.yml

```
@@ -2,48 +2,50 @@ name: Frontend - Publish release
     secrets: inherit
 
-  publish:
-    if: ${{ startsWith(github.ref_name, 'release/') }}
+  publish_libs:
     needs:
       - build_and_test_libs
-      - determine_version
     uses: ./.github/workflows/frontend_publish_to_npm.yml
     with:
-      project_version: ${{ needs.determine_version.outputs.project_version }}
+      project_version: ${{ github.env.PROJECT_VERSION }}
```

actionlint (1.7.10)

receiver of object dereference "project_version" must be type of object but got "string"

(expression)